.changeset/unlucky-starfishes-dance.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hopper-ui/styled-system": minor
+"@hopper-ui/tokens": minor
```

Comment:
It's a bit hard to see what's going on, but judging from design's message this morning, -active tokens are being removed and replaced with -selected?
If this is the case, wouldn't it be a major change?
Or did you not remove the active ones yet?

Reply:
> If this is the case, wouldn't it be a major change?

I agree on the "major" on that one, I was looking through changelog to make sure that wasn't introduced sneakily.

If I hadn't read the message in the channel last friday; I would of not assume that it would of require changes in our app.

Reply:
You are right, a renaming has been introduced post changeset. I will update the changeset. A migration path will be also provided.